utilities/sourceEvents.py

```diff
@@ -145,7 +145,10 @@ def parse(content, gmaps):
         if 'virtualEventURL' in pe:
             entry['virtualEventUrl'] = pe['virtualEventURL']
         if 'inPersonEvent' in  pe:
-            entry['isInPerson'] = pe['inPersonEvent']
+            if pe['inPersonEvent'] == "true":
```

Comment:
I think we might have to add the else case here for `isVirtualEvent` field in line 143. Currently, it looks like it ignores if the value from WebTools is false.

Reply:
what is the value in else?

Reply:
I meant that this if-else approach that you are using here might be needed for isVirtualEvent field too. Currently, for isVirtualEvent, we are setting true if the incoming value is true. I didn't see if we were setting false by default for the alternate case. Does this explain better?